pyccel/ast/numpyext.py

```diff
@@ -112,13 +115,29 @@ class NumpyFloat64(NumpyFloat):
     _precision = dtype_registry['float64'][1]
 
 #=======================================================================================
+class NumpyBool(PythonBool):
+    """ Represents a call to numpy.bool() function.
+    """
+    def __new__(cls, arg=None, base=10):
+        return super().__new__(cls, arg)
```

Comment:
Can you please add an empty line after this class definition?

This way we match the same formatting as the rest of the file.